packages/pixeloven-cli/src/extensions/pixeloven-helpers/get-arg-list.ts

```diff
@@ -0,0 +1,39 @@
+import { GluegunParameters } from "gluegun";
+
+type ArgType = "default" | "withOptions";
+
+/**
+ * Returns an arg list
+ * @description Returns an arg list from the paramaters provided to the CLI.
+ * If type eq "default"
+ *      Then: pixeloven cmd arg --option would return ["arg"]
+ * If type eq "withOptions"
+ *      Then: pixeloven cmd arg --option would return ["arg", "--option"]
+ *
+ * @param cmd
+ * @param parameters
+ * @param index
+ * @param type
+ */
+function getArgList(
+    cmd: string,
+    parameters: GluegunParameters,
+    index = 0,
+    type: ArgType = "default",
+) {
+    const args =
+        parameters.array && parameters.array.length
+            ? parameters.array.slice(index)
+            : [];
+    if (type === "withOptions" && Array.isArray(parameters.raw)) {
+        if (parameters.raw.length) {
```

Comment:
Any reason not to include this above instead of nesting since there isn't another side of this condition?